### PR TITLE
Add --pollute option to be able to pollute global space

### PR DESCRIPTION
**Pull request by mrtnzlml**
This option allows to export all variables to the `process.env.*`

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -2,93 +2,120 @@ const sinon = require('sinon')
 const fs = require('fs-extra')
 const vault2Env = require('../index')
 
-describe('vault2env', function () {
```
Comment (mrtnzlml) [inline test/index.test.js]: Oh, sorry - I removed `vault2env` label because it doesn't add anything useful...

Comment (mrtnzlml) [reply test/index.test.js]: Use this link for review: https://github.com/kiwicom/vault2env-js/pull/4/files?w=1
It's more readable with parameter `w=1`.

**Comment by jaroslav-kubicek** [comment]
LGTM, resolve conflicts so we can merge it

**Comment by jaroslav-kubicek** [comment]
looks fine, thanks!

**Pull request merged**